### PR TITLE
Improve Stripe Checkout modal experience in [register_form_stripe] shortcode #1753

### PR DESCRIPTION
**Pull request by mindctrl**
#1753

### CODE REVIEW

**Comment by mindctrl** [comment]
@tnorthcutt can you test this too? I know you've chatted with some people that want this.

**Comment by tnorthcutt** [comment]
@mindctrl I went ahead and took the liberty of pushing those changes; the main thing is calling `$.blockUI()` instead of `$('body').block()` to make sure the overlay is centered on the visible page, instead of centered on the body element.

Let me know what you think.

**Comment by mindctrl** [comment]
@tnorthcutt `$.blockUI() ` change looks good. I prefer to keep the original wording. It matches the main registration page, where I copied it from, and I think saying "processing payment" after the Stripe Checkout modal shows a checkmark and goes away feels wrong. The payment _appears_ to have been processed as soon as the modal animation does it thing and goes away, and I think we should match the perception of that. `Please wait` is more generic and less confusing.

File: includes/shortcodes.php [hunk 1]
```diff
@@ -619,3 +623,56 @@ function rcp_user_expiration_shortcode() {
...
+						} );
+					} );
+				} );
+			}
+		}
+
+		$( document ).ready( function() {
+			stripeCheckoutHelper.init();
+			$( document ).on( 'DOMNodeRemoved', '.stripe_checkout_app', function() {
+				let stripeTokenElement = document.getElementsByName( 'stripeToken' );
```
[the remark this answers is not in the document]
Comment (mindctrl) [reply includes/shortcodes.php]: Yes, we can use `let` here. All major supported browsers support it.

**Comment by tnorthcutt** [comment]
@mindctrl works for me; reverted.

I think this looks great and is ready to go. Nicely done 👏

**Comment by nosegraze** [comment]
stripe-checkout-overlay

I feel like we need to bump up the opacity on the `Please wait . . .` div. It's not very clear to read at all. It's `0.25` here and `0.5` on the main registration form. I actually think `1` looks good, but at minimum we should bump it up to `0.5`.

**Comment by mindctrl** [comment]
Agreed. I updated it and merged. Thanks @nosegraze and @tnorthcutt!

**Comment by tnorthcutt** [comment]
👏

**Pull request merged**